docs/reference/core.md

```diff
@@ -17,3 +17,9 @@ functions.  Use at your own risk.
 
 ## Versions
 ::: synapseclient.core.version_check
+
+## Remote File Storage Wrappers
```

Comment:
We could possible remove these headers or maybe wrap them like:
`<h2>Remote File Storage Wrappers</h2>`

instead. The reason I mention this is that it is duplicating some information in the Table of contents:
![image](https://github.com/Sage-Bionetworks/synapsePythonClient/assets/17128019/8eba71ba-e74c-41d0-b793-d2743598abcc)

Since `::: synapseclient.core.remote_file_storage_wrappers` also adds a header with the path of the thing we are documenting.

Any thoughts on which might look the best?

1. Keep as is
2. Replace these with the <h2> HTML tag (Removes from ToC, but keeps it on the page)
3. Remove the header as a whole

Reply:
If we want this information displayed on the `Core` page, we should leave it as is. Every other header on this page looks the same in the ToC.

Reply:
Sounds good to me!